src/Progress/CircularProgress.js

```diff
@@ -53,18 +67,37 @@ export const styleSheet = createStyleSheet('MuiCircularProgress', theme => ({
 }));
 
 function CircularProgress(props) {
-  const { classes, className, size, ...other } = props;
+  const { classes, className, size, mode, value, min, max, ...other } = props;
   const radius = size / 2;
+  const rootProps = {};
+  const svgClasses = classNames(classes.svg, {
+    [classes.indeterminateSvg]: mode === 'indeterminate',
+  });
+
+  const circleClasses = classNames(classes.circle, {
+    [classes.indeterminateCircle]: mode === 'indeterminate',
+    [classes.determinateCircle]: mode === 'determinate',
+  });
+
+  const styles = { circle: {} };
```

Comment:
If we have a single style property. I think that we could keep it flat and call it `circleStyle`.